cranelift/filetests/Cargo.toml

```diff
@@ -10,7 +10,7 @@ publish = false
 edition.workspace = true
 
 [dependencies]
-cranelift-codegen = { workspace = true, features = ["disas"] }
+cranelift-codegen = { workspace = true, features = ["disas", "zkasm"] }
```

Comment:
@nagisa , this is the part about an additional feature I was asking about. I'm still trying to figure out why other targets don't need to enable their features explicitly...

Reply:
They have got https://github.com/near/wasmtime/blob/a5e876765a938571cc1f097a78bb4ac5c5d84685/cranelift/Cargo.toml#L57 for example, and from there on I imagine it is again some sort of feature unification going on. FWIW I've just been running my tests with `cargo test --all-features` :shrug:

Reply:
This fails with
```
error[E0432]: unresolved import `cranelift_codegen::isa::zkasm`
 --> cranelift/filetests/src/test_zkasm.rs:8:9
  |
8 |     use cranelift_codegen::isa::zkasm;
  |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ no `zkasm` in `isa`
```

I think I'll leave this as is for now, but will add a comment to revise this part.